### ISSUE TITLE
These two lines are not clear

### ISSUE THREAD

**Issue by weiyuan-jiang** [GEOS-ESM/GOCART#97]
https://github.com/GEOS-ESM/GOCART/blob/8cee2066223f686bd96989a6253ee864e498b7a5/Process_Library/Chem_MieTableMod2G.F90#L654-L655

@tclune

**Comment by tclune** [comment]
Yes - this is a bug.   The intent is to restrict `rh` between 0 and 1.   You did not include the line just after but basically it should look like:
```
rh_ = max(0., min(0.99, rh))
```
(@amdasilva prefers `rh_` to `rhUse`.)
